contracts/cw-quadratic-funding/Cargo.toml

```
@@ -28,13 +28,12 @@ overflow-checks = true
 
 
 [dependencies]
-cosmwasm-std = { version = "1.0.0-beta" }
-cw-storage-plus = { version = "0.10.0", features = ["iterator"]}
+cosmwasm-std = { version = "1.1.0" }
```

We don't need the {} here.
```suggestion
cosmwasm-std = "1.1.0"
```